spec/ndx-sound.extensions.yaml

```diff
@@ -1,4 +1,100 @@
 groups:
+- neurodata_type_def: Speaker
+  neurodata_type_inc: Device
+  doc: Speaker device used for acoustic stimuli
+  datasets:
+  - name: frequency_range_in_hz
+    dtype: float
+    shape:
+    - 2
+    doc: Frequency range of the speaker in Hz
+  - name: sensitivity_in_db
+    dtype: float
+    doc: Sensitivity of the speaker in dB at 1m
+- neurodata_type_def: Microphone
+  neurodata_type_inc: Device
+  doc: Microphone device used for acoustic recordings
+  datasets:
+  - name: frequency_range_in_hz
+    dtype: float
+    shape:
+    - 2
+    doc: Frequency range of the microphone in Hz
+  - name: sensitivity_in_mv_per_pa
+    dtype: float
+    doc: Sensitivity of the microphone in mV/Pa
+- neurodata_type_def: AudioInterface
+  neurodata_type_inc: Device
+  doc: Audio interface device used for acoustic recordings
+  datasets:
+  - name: signal_to_noise_ratio_in_db
+    dtype: float
+    doc: Signal-to-noise ratio of the audio interface in dB
+  - name: channel_separation_in_db
+    dtype: float
+    doc: Channel separation of the audio interface in dB
+- neurodata_type_def: MicrophoneTable
+  neurodata_type_inc: DynamicTable
+  doc: Extends DynamicTable to hold metadata on the auditory recording system.
+  datasets:
+  - name: location
+    neurodata_type_inc: VectorData
+    dtype: text
+    shape:
+    - null
+    doc: Location of microphone.
+  - name: microphone
+    neurodata_type_inc: VectorData
+    dtype:
+      target_type: Device
+      reftype: object
+    shape:
+    - null
+    doc: Link to microphone object.
+  - name: audio_interface
+    neurodata_type_inc: VectorData
+    dtype:
+      target_type: Device
```

Comment:
shouldn't this be your newly created type?

Reply:
Yes, thx